RSSReader-Maruyama/DataModel/User.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  User.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class User: NSObject, Codable {
+    
+    let userID: String!
```

Comment:
```suggestion
    let id: String
```
idのみでも意味が通じるのでidに変更しましょう。
また！は不要です。

Reply:
修正しました。